src/instruments/src/Common/displayUnit.tsx

```diff
@@ -87,6 +88,12 @@ export const DisplayUnit: React.FC<DisplayUnitProps> = (props) => {
         return (
             <></>
         );
+    } if (homeCockpit === 1) {
+        return (
+            <>
+                <div style={{ display: state === DisplayUnitState.On ? 'block' : 'none' }}>{props.children}</div>
+            </>
+        );
     }
     return (
```

Comment:
```suggestion
    }

    if (homeCockpit === 1) {
        return (
            <>
                <div style={{ display: state === DisplayUnitState.On ? 'block' : 'none' }}>{props.children}</div>
            </>
        );
    }

    return (
```

Reply:
Formatting conflicts with another suggestion. Will manually push another update to address this.